modules/accredited_representative_portal/app/controllers/accredited_representative_portal/application_controller.rb

```diff
@@ -17,5 +19,15 @@ def verify_pilot_enabled_for_user
         raise Common::Exceptions::Forbidden, detail: message
       end
     end
+
+    def in_progress_forms_for_user
+      InProgressForm.submission_pending.where(user_uuid: @current_user.uuid)
+    end
+
+    def form_for_user(form_id)
+      return unless @current_user
```

Comment:
I don't think you need this return case. The standard `vets-api` in progress forms allow either authenticated or unauthenticated access, where your case requires authentication, right? So `@current_user` will necessarily be defined